build.gradle.kts

```diff
@@ -237,46 +238,42 @@ subprojects {
   }
 }
 
-nexusStaging {
-  packageGroup = libGroup
-  username = ossrhUsername
-  password = ossrhPassword
-}
+tasks.create<Copy>("dokkaHtmlAddIndex") {
+  outputs.upToDateWhen { false }
+
+  group = "documentation"
+  from(file("$buildDir/dokka/htmlCollector/ktx/index.html"))
+  into(file("$buildDir/dokka/htmlCollector/"))
+
+  tasks["dokkaHtmlCollector"].finalizedBy(this)
 
-val generateDocumentationIndex by tasks.registering {
   doLast {
-    val indexFile = file("$buildDir/dokka/index.html")
-    delete(indexFile)
-    indexFile.appendText("""
-      <html>
-      <head>
-        <meta charset="utf-8">
-        <title>KTX Sources Documentation</title>
-        <link rel="stylesheet" href="style.css">
-      </head>
-      <body>
-      <ul>
-        <h1>KTX Documentation</h1>
-        <p>This page contains documentation generated via Dokka from KTX sources.</p>
-        <p>To see the official KTX website, follow <a href="https://libktx.github.io/">this link</a>.</p>
-        <h2>Modules</h2>
-      ${subprojects.joinToString("\n") { "  <li><a href=\"${it.name}/\">ktx-${it.name}</a></li>" }}
-      </ul>
-      </body>
-      </html>
-      """.trimIndent())
+    file("build/dokka/htmlCollector/index.html").writeText(
+      """
+        <!DOCTYPE HTML>
+        <html lang="en-US">
+            <head>
+                <meta charset="UTF-8">
+                <meta http-equiv="refresh" content="0; url=ktx/index.html">
+                <script type="text/javascript">
+                    window.location.href = "ktx/index.html"
+                </script>
+                <title>KTX documentation</title>
+            </head>
+            <body>
+                If you are not redirected automatically, follow <a href="ktx/index.html">this link</a>.
```

Comment:
It's actually OK to link to just `ktx/` as long as the documentation is published via a HTTP server, but I'll merge this as is.